.github/workflows/continuous-integration.yml

```diff
@@ -9,11 +9,31 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        bzlmodEnabled: [true, false]
+        folder: [".", "examples"]
+        exclude:
+          # No point keeping the root workspace compatible with --noenable_bzlmod
+          - folder: "."
+            bzlmodEnabled: false
```

Comment:
did you want this to be `true`?

Reply:
this is `false` to exclude the `folder: ., bzlmodEnabled: false` entry from the matrix. It's a double negative :)